html/components/modal.stories.js

```diff
@@ -78,17 +78,30 @@ export const defaultStory = () => {
           </p>
         </div>
 
-        <footer class="sprk-o-Stack__item">
-          <button class="sprk-c-Button sprk-u-mrm">
-            Confirm
-          </button>
-
-          <button
-            class="sprk-c-Button sprk-c-Button--tertiary"
-            data-sprk-modal-cancel="exampleChoiceModal"
-          >
-            Cancel
-          </button>
+        <footer
+          class="
+            sprk-o-Stack__item
+            sprk-c-Modal__footer
+            sprk-o-Stack
+            sprk-o-Stack--split@xs
+            sprk-o-Stack--end-row
+            sprk-o-Stack--medium">
+          <div class="sprk-o-Stack__item">
+            <button
```

Comment:
```suggestion
            <button
              sprkButton
```
We just want to match the story file to what we are doing internally too! Sorry this is extra work but since you went through all the work internally we'll wanna match! Ty!